[2025-06-27T04:51:55.616+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-27T04:51:55.713+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ga4_raw_data_elt_dev.create_ga4_events_dataset scheduled__2025-06-05T02:10:00+00:00 [queued]>
[2025-06-27T04:51:55.814+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ga4_raw_data_elt_dev.create_ga4_events_dataset scheduled__2025-06-05T02:10:00+00:00 [queued]>
[2025-06-27T04:51:55.815+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-06-27T04:51:55.924+0000] {taskinstance.py:2888} INFO - Executing <Task(BigQueryCreateEmptyDatasetOperator): create_ga4_events_dataset> on 2025-06-05 02:10:00+00:00
[2025-06-27T04:51:55.947+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3660) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-27T04:51:55.953+0000] {standard_task_runner.py:72} INFO - Started process 3714 to run task
[2025-06-27T04:51:55.961+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ga4_raw_data_elt_dev', 'create_ga4_events_dataset', 'scheduled__2025-06-05T02:10:00+00:00', '--job-id', '913', '--raw', '--subdir', 'DAGS_FOLDER/GA4_data_elt_dev.py', '--cfg-path', '/tmp/tmp8wco4fs7']
[2025-06-27T04:51:55.980+0000] {standard_task_runner.py:105} INFO - Job 913: Subtask create_ga4_events_dataset
[2025-06-27T04:51:56.159+0000] {task_command.py:467} INFO - Running <TaskInstance: ga4_raw_data_elt_dev.create_ga4_events_dataset scheduled__2025-06-05T02:10:00+00:00 [running]> on host f79cc72a155d
[2025-06-27T04:51:56.732+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ga4_raw_data_elt_dev' AIRFLOW_CTX_TASK_ID='create_ga4_events_dataset' AIRFLOW_CTX_EXECUTION_DATE='2025-06-05T02:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-05T02:10:00+00:00'
[2025-06-27T04:51:56.781+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-27T04:51:57.009+0000] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-06-27T04:51:57.057+0000] {base.py:84} INFO - Retrieving connection 'google_cloud_default'
[2025-06-27T04:51:57.069+0000] {bigquery.py:533} INFO - datasetId was not specified in `dataset_reference`. Will use default value ga4_events_dev.
[2025-06-27T04:51:57.073+0000] {bigquery.py:533} INFO - projectId was not specified in `dataset_reference`. Will use default value sixth-topic-349709.
[2025-06-27T04:51:57.078+0000] {bigquery.py:544} INFO - Creating dataset: ga4_events_dev in project: sixth-topic-349709 
[2025-06-27T04:51:59.146+0000] {bigquery.py:548} INFO - Dataset created successfully.
[2025-06-27T04:51:59.564+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-27T04:51:59.582+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ga4_raw_data_elt_dev, task_id=create_ga4_events_dataset, run_id=scheduled__2025-06-05T02:10:00+00:00, execution_date=20250605T021000, start_date=20250627T045155, end_date=20250627T045159
[2025-06-27T04:51:59.762+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-27T04:52:00.015+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-06-27T05:36:21.097+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-27T05:36:21.162+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ga4_raw_data_elt_dev.create_ga4_events_dataset scheduled__2025-06-05T02:10:00+00:00 [queued]>
[2025-06-27T05:36:21.172+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ga4_raw_data_elt_dev.create_ga4_events_dataset scheduled__2025-06-05T02:10:00+00:00 [queued]>
[2025-06-27T05:36:21.176+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-06-27T05:36:21.241+0000] {taskinstance.py:2888} INFO - Executing <Task(BigQueryCreateEmptyDatasetOperator): create_ga4_events_dataset> on 2025-06-05 02:10:00+00:00
[2025-06-27T05:36:21.251+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=118) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-27T05:36:21.260+0000] {standard_task_runner.py:72} INFO - Started process 157 to run task
[2025-06-27T05:36:21.263+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ga4_raw_data_elt_dev', 'create_ga4_events_dataset', 'scheduled__2025-06-05T02:10:00+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/GA4_data_elt_dev.py', '--cfg-path', '/tmp/tmp0oiha7hn']
[2025-06-27T05:36:21.276+0000] {standard_task_runner.py:105} INFO - Job 24: Subtask create_ga4_events_dataset
[2025-06-27T05:36:21.471+0000] {task_command.py:467} INFO - Running <TaskInstance: ga4_raw_data_elt_dev.create_ga4_events_dataset scheduled__2025-06-05T02:10:00+00:00 [running]> on host ba0a3a3d8846
[2025-06-27T05:36:21.670+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ga4_raw_data_elt_dev' AIRFLOW_CTX_TASK_ID='create_ga4_events_dataset' AIRFLOW_CTX_EXECUTION_DATE='2025-06-05T02:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-05T02:10:00+00:00'
[2025-06-27T05:36:21.695+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-27T05:36:21.735+0000] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-06-27T05:36:21.741+0000] {base.py:84} INFO - Retrieving connection 'google_cloud_default'
[2025-06-27T05:36:21.742+0000] {bigquery.py:533} INFO - datasetId was not specified in `dataset_reference`. Will use default value ga4_events_dev.
[2025-06-27T05:36:21.744+0000] {bigquery.py:533} INFO - projectId was not specified in `dataset_reference`. Will use default value sixth-topic-349709.
[2025-06-27T05:36:21.746+0000] {bigquery.py:544} INFO - Creating dataset: ga4_events_dev in project: sixth-topic-349709 
[2025-06-27T05:36:22.910+0000] {bigquery.py:548} INFO - Dataset created successfully.
[2025-06-27T05:36:22.970+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-27T05:36:22.972+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ga4_raw_data_elt_dev, task_id=create_ga4_events_dataset, run_id=scheduled__2025-06-05T02:10:00+00:00, execution_date=20250605T021000, start_date=20250627T053621, end_date=20250627T053622
[2025-06-27T05:36:23.001+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-27T05:36:23.055+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-27T05:36:23.057+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-06-27T05:39:29.289+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-27T05:39:29.350+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ga4_raw_data_elt_dev.create_ga4_events_dataset scheduled__2025-06-05T02:10:00+00:00 [queued]>
[2025-06-27T05:39:29.360+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ga4_raw_data_elt_dev.create_ga4_events_dataset scheduled__2025-06-05T02:10:00+00:00 [queued]>
[2025-06-27T05:39:29.362+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-06-27T05:39:29.381+0000] {taskinstance.py:2888} INFO - Executing <Task(BigQueryCreateEmptyDatasetOperator): create_ga4_events_dataset> on 2025-06-05 02:10:00+00:00
[2025-06-27T05:39:29.394+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=123) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-27T05:39:29.398+0000] {standard_task_runner.py:72} INFO - Started process 169 to run task
[2025-06-27T05:39:29.409+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ga4_raw_data_elt_dev', 'create_ga4_events_dataset', 'scheduled__2025-06-05T02:10:00+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/GA4_data_elt_dev.py', '--cfg-path', '/tmp/tmp1jairk6f']
[2025-06-27T05:39:29.421+0000] {standard_task_runner.py:105} INFO - Job 30: Subtask create_ga4_events_dataset
[2025-06-27T05:39:29.522+0000] {task_command.py:467} INFO - Running <TaskInstance: ga4_raw_data_elt_dev.create_ga4_events_dataset scheduled__2025-06-05T02:10:00+00:00 [running]> on host 616f50362410
[2025-06-27T05:39:29.660+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ga4_raw_data_elt_dev' AIRFLOW_CTX_TASK_ID='create_ga4_events_dataset' AIRFLOW_CTX_EXECUTION_DATE='2025-06-05T02:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-05T02:10:00+00:00'
[2025-06-27T05:39:29.664+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-27T05:39:29.694+0000] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-06-27T05:39:29.698+0000] {base.py:84} INFO - Retrieving connection 'google_cloud_default'
[2025-06-27T05:39:29.701+0000] {bigquery.py:533} INFO - datasetId was not specified in `dataset_reference`. Will use default value ga4_events_dev.
[2025-06-27T05:39:29.701+0000] {bigquery.py:533} INFO - projectId was not specified in `dataset_reference`. Will use default value sixth-topic-349709.
[2025-06-27T05:39:29.704+0000] {bigquery.py:544} INFO - Creating dataset: ga4_events_dev in project: sixth-topic-349709 
[2025-06-27T05:39:31.063+0000] {bigquery.py:548} INFO - Dataset created successfully.
[2025-06-27T05:39:31.340+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-27T05:39:31.343+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ga4_raw_data_elt_dev, task_id=create_ga4_events_dataset, run_id=scheduled__2025-06-05T02:10:00+00:00, execution_date=20250605T021000, start_date=20250627T053929, end_date=20250627T053931
[2025-06-27T05:39:31.443+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-27T05:39:31.596+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-27T05:39:31.611+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-06-27T05:48:39.197+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-27T05:48:39.290+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ga4_raw_data_elt_dev.create_ga4_events_dataset scheduled__2025-06-05T02:10:00+00:00 [queued]>
[2025-06-27T05:48:39.315+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ga4_raw_data_elt_dev.create_ga4_events_dataset scheduled__2025-06-05T02:10:00+00:00 [queued]>
[2025-06-27T05:48:39.317+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-06-27T05:48:39.408+0000] {taskinstance.py:2888} INFO - Executing <Task(BigQueryCreateEmptyDatasetOperator): create_ga4_events_dataset> on 2025-06-05 02:10:00+00:00
[2025-06-27T05:48:39.469+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=180) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-27T05:48:39.476+0000] {standard_task_runner.py:72} INFO - Started process 217 to run task
[2025-06-27T05:48:39.507+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ga4_raw_data_elt_dev', 'create_ga4_events_dataset', 'scheduled__2025-06-05T02:10:00+00:00', '--job-id', '38', '--raw', '--subdir', 'DAGS_FOLDER/GA4_data_elt_dev.py', '--cfg-path', '/tmp/tmpwt4zq037']
[2025-06-27T05:48:39.549+0000] {standard_task_runner.py:105} INFO - Job 38: Subtask create_ga4_events_dataset
[2025-06-27T05:48:39.768+0000] {task_command.py:467} INFO - Running <TaskInstance: ga4_raw_data_elt_dev.create_ga4_events_dataset scheduled__2025-06-05T02:10:00+00:00 [running]> on host 546ad4141aeb
[2025-06-27T05:48:40.105+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ga4_raw_data_elt_dev' AIRFLOW_CTX_TASK_ID='create_ga4_events_dataset' AIRFLOW_CTX_EXECUTION_DATE='2025-06-05T02:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-05T02:10:00+00:00'
[2025-06-27T05:48:40.134+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-27T05:48:40.164+0000] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-06-27T05:48:40.171+0000] {base.py:84} INFO - Retrieving connection 'google_cloud_default'
[2025-06-27T05:48:40.178+0000] {bigquery.py:533} INFO - datasetId was not specified in `dataset_reference`. Will use default value ga4_events_dev.
[2025-06-27T05:48:40.179+0000] {bigquery.py:533} INFO - projectId was not specified in `dataset_reference`. Will use default value sixth-topic-349709.
[2025-06-27T05:48:40.180+0000] {bigquery.py:544} INFO - Creating dataset: ga4_events_dev in project: sixth-topic-349709 
[2025-06-27T05:48:41.434+0000] {bigquery.py:548} INFO - Dataset created successfully.
[2025-06-27T05:48:41.539+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-27T05:48:41.541+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ga4_raw_data_elt_dev, task_id=create_ga4_events_dataset, run_id=scheduled__2025-06-05T02:10:00+00:00, execution_date=20250605T021000, start_date=20250627T054839, end_date=20250627T054841
[2025-06-27T05:48:42.049+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-27T05:48:42.228+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-27T05:48:42.246+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
